examples/search.js

```diff
@@ -16,6 +16,16 @@ goog.require('goog.asserts');
 goog.require('ngeo.map.module');
 goog.require('ngeo.search.module');
 
+// import './search.css';
+// import 'jquery';
+// import 'angular';
+// import 'angular-gettext';
+// import 'angular-ui-date';
+// import 'angular-float-thead';
+// import 'floatthead';
+// import 'corejs-typeahead';
+// import '../utils/watchwatchers.js';
```

Comment:
Let's create a examples/common_dependencies.js file with:
+// import 'jquery';
+// import 'angular';
+// import '../utils/watchwatchers.js';
node_modules/bootstrap/dist/js/bootstrap.js',

And remove the other imports since they should come by dependency anyway.